kolibri/plugins/management/assets/src/state/facilityConfigPageActions.js

```diff
@@ -0,0 +1,88 @@
+/* eslint-disable prefer-arrow-callback */
+const omit = require('lodash/fp/omit');
+const {
+  FacilityResource,
+  FacilityDatasetResource,
+} = require('kolibri').resources;
+const ConditionalPromise = require('kolibri.lib.conditionalPromise');
+const { samePageCheckGenerator } = require('kolibri.coreVue.vuex.actions');
+const preparePage = require('./preparePage');
+const { PageNames, defaultFacilityConfig, notificationTypes } = require('../constants');
+
+// When app is installed, the Facility is assigned id of `1`. Hardcoded here for now.
+const FACILITY_ID = 1;
```

Comment:
Instead of hard-coding the facility ID, we might want to grab it from the user's session object?

The only caveat is that device superusers don't have an associated facility, which would limit this functionality facility admins only (which is also true of some other facility-specific functionality like coach tools)